<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Usuario</class>
 <widget class="QDialog" name="Usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>414</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Nombre</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_Nombre">
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Apellido</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Apellido</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_Apellido">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CUIL</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_CUIL</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_CUIL">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ingrese su &lt;span style=&quot; font-weight:600;&quot;&gt;edad&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="inputMask">
      <string>99-99999999-9; </string>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Fecha de nacimiento</string>
     </property>
     <property name="buddy">
      <cstring>dateEdit_Nacim</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDateEdit" name="dateEdit_Nacim">
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="displayFormat">
      <string>d/M/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBox_Habilitado">
     <property name="text">
      <string>Habilitado</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Calificaci√≥n</string>
     </property>
     <widget class="QProgressBar" name="Calif_progressBar">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>55</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QDial" name="Calif_dial">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>100</width>
        <height>74</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDial
{
	background-color: blue;
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Calif_lcdNumber">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: black;
color: red;
border: 2px solid red;</string>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="numDigits">
       <number>5</number>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Sexo</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_M">
        <property name="text">
         <string>Masculino</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_F">
        <property name="text">
         <string>Femenino</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+H</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Estado civil</string>
     </property>
     <property name="buddy">
      <cstring>comboBox_ECivil</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="comboBox_ECivil">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Soltero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Casado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Divorciado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Viudo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En concubinato</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/images/emule.png</normalon>
       </iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Nombre completo</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_NombComp</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_NombComp">
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Calif_dial</tabstop>
  <tabstop>radioButton_M</tabstop>
  <tabstop>radioButton_F</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Usuario</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Usuario</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Calif_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Calif_lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Calif_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Calif_progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Nombre</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Usuario</receiver>
   <slot>nombreChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Apellido</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Usuario</receiver>
   <slot>nombreChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nombreChanged(QString)</slot>
 </slots>
</ui>
